utils/flare.py

```diff
@@ -3,9 +3,12 @@
 import cStringIO as StringIO
 from functools import partial
 import glob
+import grp
 import logging
-import os.path
+import os
+import pwd
```

Comment:
Is it possible to `import` `grp` and `pwd` on Windows without error ? (I saw that they were Unix-only modules)